osfoffline/polling_osf_manager/polling.py

```diff
@@ -110,48 +110,21 @@ def make_local_remote_tuple_list(self, local_list, remote_list):
         assert None not in local_list
         assert None not in remote_list
 
-        combined_list = local_list + remote_list
-        sorted_combined_list = sorted(combined_list, key=self.get_id)
+        local_files = {}
```

Comment:
Himanshu has approved this rewrite, but we'll be sure to test the jeepers out of it. :shipit: